doc/source/building/index.rst

```diff
@@ -203,12 +203,26 @@ your system.
         invoking a Fortran compiler in the shell you use (e.g., ``gfortran
         --version`` or ``ifort --version``).
 
+  .. warning::
+
+       When using a conda environment it is possible that the environment
+       creation will not work due to an outdated Fortran compiler. If that
+       happens, remove the ``compilers`` entry from ``environment.yml`` and
+       try again. The Fortran compiler should be installed as described in
+       this section.
```

Comment:
I can't check because the html didn't render, but this seems like the wrong place. The warning is specific to conda-on-Windows, but given the indentation level it may now always show up?

Reply:
Also, it removes "on Windows", and I don't understand why. It should be Windows-specific.

Reply:
Should be fixed now. I had an indentation issue.